processor/processor_test.go

```diff
@@ -2258,11 +2258,18 @@ var _ = Describe("Processor", Ordered, func() {
 			m := processor.getNonSuccessfulMetrics(transformerResponse,
 				&commonMetadata,
 				eventsByMessageID,
-				transformer.DestTransformerStage,
-				false,
-				false)
+				types.EVENT_FILTER,
+				types.DEST_TRANSFORMER,
+			)
 
-			key := fmt.Sprintf("%s!<<#>>!%s!<<#>>!%s!<<#>>!%s!<<#>>!%s", commonMetadata.SourceID, commonMetadata.DestinationID, commonMetadata.SourceJobRunID, commonMetadata.EventName, commonMetadata.EventType)
+			key := fmt.Sprintf(
+				"%s!<<#>>!%s!<<#>>!%s!<<#>>!%s!<<#>>!%s",
+				commonMetadata.SourceID,
+				commonMetadata.DestinationID,
+				commonMetadata.SourceJobRunID,
+				commonMetadata.EventName,
+				commonMetadata.EventType,
+			)
```

Comment:
[minor] you can also do:
```suggestion
			key := strings.Join([]string{
				commonMetadata.SourceID,
				commonMetadata.DestinationID,
				commonMetadata.SourceJobRunID,
				commonMetadata.EventName,
				commonMetadata.EventType,
			},  "!<<#>>!")
```